medical/models/medical_patient.py

```diff
@@ -101,14 +101,9 @@ def _create_vals(self, vals):
         })
         return vals
 
-    @api.model
-    def _get_default_image(self, vals):
-        res = super(MedicalPatient, self)._get_default_image(vals)
-        if res:
-            return res
-        img_path = get_module_resource(
+    @api.model_cr_context
+    def _get_default_image_path(self, vals):
+        super(MedicalPatient, self)._get_default_image_path(vals)
```

Comment:
Removing it breaks the inheritance chain, which is a not a good thing IMO. So while yes, it is 100% useless given knowledge we have about the architecture of the system as it stands before us now, this may not always be the case once other module children start coming to play.